sources/packages/web/src/components/aest/institution/modals/ApproveDenyDesignationModal.vue

```diff
@@ -26,7 +26,7 @@
                 variant="outlined"
                 :rules="[checkStringDateFormatRule]"
                 hide-details="auto" /></v-col></v-row
-          ><v-divider></v-divider>
+          ><v-divider class="border-opacity-100" :thickness="2" />
```

Comment:
Just to think about it, should we consider that the project will use the opacity 100 as a default which means that we can override the default v-divider opacity globally?

Reply:
In that case(if border-opacity-100 is going to be set for horizontal and vertical divider) we can do like this to set the default class. 

![image](https://github.com/bcgov/SIMS/assets/54600590/86dbaa31-9995-4101-b6d0-a8425d76e35e)

Reply:
I agree we can have it globally

Reply:
We can just change the CSS:

![image](https://github.com/bcgov/SIMS/assets/78114138/eb0d2b8d-c521-4c39-9d41-ca13440245b7)